doc/js-api/caching.rst

```diff
@@ -0,0 +1,46 @@
+Caching
+=======
```

Comment:
I think an introductory note that is initialized as `remoteStorage.caching`, but only if one doesn't configure `caching: false`, would be useful.